doc/source/crc.md

```diff
@@ -0,0 +1,118 @@
+# Install on OpenShift
+
+## Create a project / namespace
+
+Create your project using the `oc` command line.
+
+```bash
+oc new-project <project-name>
+```
+
+## Configure webhooks
+
+Edit the config YAML in the `config` directory of your cloned repo:
+
```

Comment:
no instructs for cloning repo?

Reply:
I'll add a TODO in the crc.md.   I'd like to commit these notes but then I need to fix or hide it ASAP.  Will do a separate PR for the config files.